CHANGELOG.md

```diff
@@ -11,6 +11,77 @@ Some experimental features can be enabled / disabled using [Feature Gates](docs/
 
 ## Unreleased
 
+## 1.0.0 - 2021-04-09
+
+Includes all the changes from [0.13.1].
+
+The AntreaPolicy feature is graduated from Alpha to Beta and is therefore enabled by default.
+
+### Added
+
+- Add [Egress] feature to configure SNAT policies for Pod-to-external traffic. [Alpha - Feature Gate: `Egress`]
```

Comment:
Question - do we need to list each individual changes and PRs in change log?

Reply:
I feel like this is our main new feature for this release, and the current version strikes a good balance in terms of giving credit / providing information about the feature, without being *too* verbose

Reply:
Ok. I feel documentation PRs are not necessary, and description for datapath changes can be shorter, but would let you decide.